parambokeh/__init__.py

```diff
@@ -214,6 +214,9 @@ def change_event(self):
         if isinstance(p_obj, param.Range):
             new_values = tuple(new_values)
 
+        if isinstance(w, CheckboxGroup):
+            new_values = True if (len(new_values)>0 and new_values[0]==0) else False
```

Comment:
Just to clarify, the code isn't meant to "Set boolean parameter from list of zeros", it's really meant to "Set boolean parameter depending on whether the index of that parameter is in the indicated list".  The list in this case should never have more than one element, but if it did, it would be something like ``[0,1,2]`` *not* ``[0,0,0]``.

Reply:
(The code is correct; it's just the commit message that's confusing.)